examples/oft-aptos/aptos.layerzero.config.ts

```diff
@@ -0,0 +1,161 @@
+import { EndpointId } from '@layerzerolabs/lz-definitions-v3'
+import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities-v3'
```

Comment:
Why are we adding "-v3'" at the end here?

This should be replaced with the standard:

```
layerzerolabs/lz-v2-utilities
@layerzerolabs/lz-definitions
```